base/process.jl

```diff
@@ -702,7 +702,7 @@ function arg_gen(head, tail...)
     tail = arg_gen(tail...)
     vals = String[]
     for h = head, t = tail
-        push!(vals, cstr(bytestring(h, t)))
+        push!(vals, cstr(String(h*t)))
```

Comment:
Isn't this going to be less efficient than the original? You're creating a string with `*`, and then potentially converting it to `String`. Anyway, `cstr` calls `String`, so you can drop the call here.